packages/web/src/executors/file-server/file-server.impl.ts

```diff
@@ -1,12 +1,28 @@
+import { ExecutorContext, joinPathFragments, logger } from '@nrwl/devkit';
+import { workspaceLayout } from '@nrwl/workspace/src/core/file-utils';
 import { exec, execSync } from 'child_process';
-import { ExecutorContext, joinPathFragments } from '@nrwl/devkit';
-import ignore from 'ignore';
-import { readFileSync } from 'fs';
-import { Schema } from './schema';
 import { watch } from 'chokidar';
-import { workspaceLayout } from '@nrwl/workspace/src/core/file-utils';
+import { existsSync, readFileSync } from 'fs';
+import ignore from 'ignore';
+import { Schema, ProxyConfig } from './schema';
+import { join } from 'path';
 
-function getHttpServerArgs(options: Schema) {
+// Find and import proxyConfig configuration if supplied
+function getProxyConfig(proxyConfigPath, context) {
+  let proxyConfig: ProxyConfig;
+
+  const fullProxyConfigPath = proxyConfigPath
+    ? join(context.root, proxyConfigPath)
+    : join(context.root, 'proxy.conf.json');
```

Comment:
I was following a convention similar to the server executor here. I'm not sure if defaulting to `proxy.conf.json`is the way to go (name collision with the server executor). Thoughts?

Reply:
Sorry, I misunderstood the intent here. Please change it to an object `proxyOption: { secure: boolean }`.